webpage_of_me/style.css

```diff
@@ -0,0 +1,303 @@
+    border-radius: 10%;
+}
+.dimm {
+    background:linear-gradient(to bottom, rgba(15,39,46,0.5), rgba(245,245,245,.95));
+    margin: 0 auto .5rem auto;
+    width:90%;
+    border-radius: 10%;
+    margin-top: 1rem;
+}
+.photo_me {
+    border-radius: 40%;
+    width: 90%;
+    max-width: 400px;
+    /*removed the need to specify different percentages at diff breakpoints*/
+    display: block;
+    margin: 0 auto;
+    padding: 1rem 0;
+    opacity: 1;
+    background-image: url(images/raccoon_dog.jpg) no-repeat fixed center;
+}
+/* .tinted_project {
+    background:linear-gradient(to bottom, rgba(15,39,46,0.5), rgba(245,245,245,.95)), url("images/DogAdoption.png");
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin: 0 auto .5rem auto;
+    width:90%;
+    height:20rem;
+} */
+/*Images for tablets*/
+@media screen and (min-width: 500px) and (max-width: 768px) {
+    .photo_me {
+        border-radius: 20%;
+        display: inline-block;
+        /* width: 65%; like here*/
+        padding: .75rem ;
+    }
+}
+/*Images for Laptop screens and more*/
+@media screen and (min-width: 768px) {
+    .photo_me {
+        border-radius: 20%;
+        display: inline-block;
+        /* width: 45%; and like here */
+        padding: .75rem ;
+        vertical-align:top;
+    }
+}
+/*formatting for the navigation bar*/
+.navigation ul {
+    list-style-type: none;
+    margin: 1rem 0 1rem 0;
+    text-align: center;
+}
+.navigation li {
+    display: inline-block;
+    width: 30%;
+}
+.navigation a {
+    display: block;
+    text-align: center;
+    background-color: hsl(193, 51%, 42%);
+    text-decoration: none;
+    padding: 0.35rem;
+    color: hsl(220, 1%, 96%);
+    border-radius: .5rem;
+    font-weight:800;
+}
+.selected {
+    background-color: hsl(193, 48%, 61%);
+    color: white;
+}
+li a:hover {   
+    background-color:hsl(194, 20%, 63%);
+    font-size:1rem;
+} 
+.navigation  {
+    margin: 0 auto;
+    max-width: 90%;
+}
+
+/*Picture Nav Links*/
+.icon_nav {
+    display:block;
+    text-align:center;
+    padding: 1rem 0;
+}
+.icon_nav a{
+    width:20%;
+    display: inline-block;
+}
+.icon {
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center;
+    width:100%;
+    height:50px;
+}
+.coroflot {
+    background-image: url("images/iconmonstr-briefcase-5-240.png");
+}
+.linkedin {
+    background-image: url("images/iconmonstr-linkedin-3-240.png");
+}
+.github{
+    background-image: url("images/iconmonstr-github-3-240.png");
+}
+.email{
+    background-image: url("images/iconmonstr-email-10-240.png");
+}
+@media screen and (max-width: 396px) {
+    .navigation li {
+        display: block;
+        width: 100%;
+    }
+    .icon {
+        height:40px
+    }
+    .nav_button {
+        padding: .1rem 0;
+    }
+}
+/*FOUR K RESOLUTION*/
+@media screen and (min-width: 2560px) {
+    .icon {
+        height:100px;
+    }
+}
```

Comment:
You can combine all these rule sets under a single media-query. I realize you may not have known you were repeating it -- it is super easy to lose track of all the rules in a long stylesheet -- but just note it here if you decide to go back and add more content to your site.